src/main/java/com/deusgmbh/xcusatio/data/lecturer/LecturerManager.java

```diff
@@ -30,13 +30,118 @@ public LecturerManager() {
                 .addLecture("Logik")
                 .addLecture("Lineare Algebra")
                 .addLecture("Analysis")
+                .addLecture("Algorithmen")
                 .addTag(Tag.FUNNY)
                 .addTag(Tag.HIGH_TRAFFIC))
-                .add(new Lecturer("Kruse").addLecture("C")
-                        .addLecture("Softwareengineering 1")
-                        .addLecture("Softwareengineering 2")
+                .add(new Lecturer("Kruse").addLecture("Programmieren in C")
+                        .addLecture("Grundlagen des Softwareengineering 1")
+                        .addLecture("Grundlagen des Softwareengineering 2")
                         .addTag(Tag.FUNNY)
-                        .addTag(Tag.SUCK_UP));
+                        .addTag(Tag.SUCK_UP))
+                .add(new Lecturer("Lohnert").addLecture("Betriebssysteme")
+                        .addTag(Tag.ACCIDENT)

```

Comment:
Bitte nur Tags zu einem lecturer hinzufügen, wenn die auch einen true wert für "lecturerpreference" haben.
Und ich meine Accident hat das nicht.

Reply:
Also einmal alle überarbeiten. 
Den jeder Tag den ein lecturer hat wird automatisch in die tag liste für den context geschrieben. Bei Accident würde dann eine ausrede generiert werden, die sonst nur annehmbar wäre, wenn es wirklich einen unfall gab. Das gilt auch für alle anderen ausreden